### ISSUE TITLE
MutationObserver.observe() should not have optional second argument

### ISSUE THREAD

**Issue by diesal11** [microsoft/TypeScript-DOM-lib-generator#982]
According to [MutationObserver MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver/observe), calling `.observe()` without an options argument will throw a type error.

The docs for the options type MutationObserverInit state that:
```
At a minimum, one of childList, attributes, and/or characterData must be true when you call observe(). Otherwise, a TypeError exception will be thrown.
```

However, all of childList, attributes & characterData are false by default, including when an options argument is not provided.  Therefore i believe that the secondary argument for observe() being optional is incorrect.

**Comment by felipe-sme** [comment]
Yep, Typescript marks the `MutationObserver` observe method signature as `MutationObserver.observe(target: Node, options?: MutationObserverInit): void`. Calling it without an options object doesn't give you a static error, but you'll get a runtime type error as [it isn't optional](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver/observe#options). At least one of the properties needs to be passed in as true.
